src/pages/PeoplePage.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { useEffect, useState } from 'react';
+
+import { Person } from '../types';
+import { getPeople } from '../api';
+import { PeopleTable } from '../components/PeopleTable';
+import { Loader } from '../components/Loader';
+
+export const PeoplePage: React.FC = () => {
+  const [peopleFromServer, setPeopleFromServer] = useState<Person[]>([]);
+  const [error, setError] = useState(false);
+  const [isLoading, setIsLoading] = useState(true);
+
+  useEffect(() => {
+    getPeople()
+      .then((people) => setPeopleFromServer(people))
+      .catch(() => setError(true))
+      .finally(() => setIsLoading(false));
+  }, []);
+
+  let content;
+
+  if (error) {
```

Comment:
This logic looks so interesting. Maybe you also can use ```switch-case```, but I'm not sure about this logic

Reply:
@ivaZaiets i'm agree with Ivanna, let's use `switch` in this case